docs/backend-system/building-backends/08-migrating.md

```diff
@@ -754,3 +754,65 @@ going easily, but feel free to move it out to where it fits best. As you migrate
 your entire plugin flora to the new backend system, you will probably make more
 and more of these modules as "first class" things, living right next to the
 implementations that they represent, and being exported from there.
+
+### The Auth Plugin
+
+A basic installation of the events plugin looks as follows.
+
+```ts title="packages/backend/src/index.ts"
+import { createBackend } from '@backstage/backend-defaults';
+
+const backend = createBackend();
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-auth-backend'));
+backend.add(import('@backstage/plugin-auth-backend-module-github-provider'));
+/* highlight-add-end */
+
+backend.start();
+```
+
+You should already have `@backstage/plugin-auth-backend` installed from your previous backend. In the example above we're installing the GithHub module, but you'll need to install your respective authentication provider module.
+
+> If you are not migrating from an existing backend to the new backend system, you'll need to [configure your authentication provider](https://backstage.io/docs/auth/#configuring-authentication-providers) in your `app-config.yaml`.
+
+#### Using Legacy Providers
```

Comment:
Noting the @awanlin is working on docs for the new auth backend that we should add a link to.

Reply:
Sorry for the slow reply @minkimcello, I would say the what you have under "The Auth Plugin" has been covered but I think the "Using Legacy Providers" could go right under the "Custom Resolver" section here: https://backstage.io/docs/backend-system/building-backends/migrating#custom-resolver